Transmission.xcodeproj/project.pbxproj

```diff
@@ -4496,6 +4496,11 @@
 				);
 				INFOPLIST_FILE = "macosx/QuickLookPlugin/QuickLookPlugin-Info.plist";
 				INSTALL_PATH = /Library/QuickLook;
+				LD_RUNPATH_SEARCH_PATHS = (
```

Comment:
I don't see why we need this, it's not like we are bundling any new dynamic libraries as part of this PR.
Might be a leftover after enabling ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES.
We should be able to safely revert this.